pkg/fieldpath/fieldpath.go

```diff
@@ -85,9 +85,17 @@ func ExtractContainerResourceValue(fs *api.ResourceFieldSelector, container *api
 
 	switch fs.Resource {
 	case "limits.cpu":
-		return convertResourceCPUToString(container.Resources.Limits.Cpu(), divisor)
+		cpuLimits := *container.Resources.Limits.Cpu()
```

Comment:
ignore my comment, I just noticed that you are assigning a value from the map allocatable below.